pds4indextools/index_label_template_pds.xml

```diff
@@ -0,0 +1,178 @@
+            <comment></comment>                                                                                         
+        </Source_Product_Internal>
+        $END_FOR
+        $END_IF
+        $IF(Source_Product_External)
+        $FOR(field, k=Source_Product_External)
+        <Source_Product_External>                                
+            <external_source_product_identifier></external_source_product_identifier>                                                    
+            <reference_type></reference_type>                                           
+            <doi></doi>                                                                                                                
+            <curating_facility></curating_facility>
+            <description></description>
+        </Source_Product_External>
+        $END_FOR
+        $END_IF
+    </Reference_List>
+    $IF(Product_Ancillary)
+    <File_Area_Ancillary>
+    $END_IF
+    $IF(Product_Metadata_Supplemental)
+    <File_Area_Metadata>
+    $END_IF                                   
+        <File>
+            <file_name>$BASENAME(TEMPFILE)$</file_name>                                              
+            <local_identifier>index-table</local_identifier>              
+            <creation_date_time>$DATETIME(creation_date_time)$</creation_date_time>                        
+            <md5_checksum>$FILE_MD5(TEMPFILE)$</md5_checksum>                                       
+            <comment></comment>                                           
+        </File>
+        <Header>                                                                       
+            <offset unit="byte">0</offset>
+            <object_length unit="byte">$object_length_h$</object_length>                                            
+            <parsing_standard_id>UTF-8 Text</parsing_standard_id>                
+            <description>Provides the column headers, separated by commas, for the data table.</description>
+        </Header>
+        $IF(Table_Character)
+        <Table_Character>
+            <offset unit="byte"></offset>      
+            <object_length unit="byte">$object_length_t$</object_length>                                        
+            <records>$FILE_RECORDS(TEMPFILE)$</records> 
+            <record_delimiter>Line-Feed</record_delimiter>
+            <description></description>                                           
+            <Record_Character>
+                <fields>$fields$</fields> 
+                <groups>0</groups>
+                <record_length unit="byte"></record_length>                        
+                $FOR(field, k=Field_Content)
+                    <Field_Character>
+                        <name>$field['name']$</name>
+                        <field_number>$k + 1$</field_number>
+                        <field_location unit="byte">$field['field_location']$</field_location>                     
```

Comment:
I think `field_location` and `field_length` need to be omitted if the table is not fixed-width.

Reply:
There is a check during label generation whether args.fixed_width was used. If it was, it uses Field_Character. If it was not used, it will use Field_Delimited, which excludes these values.

Reply:
Well, it's not working. I'm not using fixed-width and I'm getting `Field_Character` in my label.

Reply:
@rfrenchseti I found the issue. The template wasn't the problem. It was the default_values.yaml file. Table_Character was set to True for some reason.